.circleci/run.sh

```diff
@@ -2,7 +2,7 @@
 
 set -uexo pipefail
 
-HOST_DMD_VER=2.072.2 # same as in dmd/src/posix.mak
+HOST_DMD_VER=2.074.1 # same as in dmd/src/posix.mak
```

Comment:
FYI: auto-tester needs to be bumped too, see the [discussion on the previous PR](https://github.com/dlang/dmd/pull/6907#issuecomment-309107255) and the [NG discussion](http://forum.dlang.org/post/bjgyigddweededitvxnz@forum.dlang.org) - especially [Martin's reply](http://forum.dlang.org/post/zbxaviowooiaenaawmgs@forum.dlang.org).